### PR TITLE
feat: add image transforms

### PR DESCRIPTION
**Pull request by inian**

### CODE REVIEW

File: internal/utils/misc.go [hunk 1]
```diff
@@ -27,25 +27,27 @@ const (
...
-	DenoRelayImage = "supabase/deno-relay:v1.5.0"
+	KongImage       = "library/kong:2.8.1"
+	InbucketImage   = "inbucket/inbucket:3.0.3"
+	PostgrestImage  = "postgrest/postgrest:v9.0.1.20220717"
+	DifferImage     = "supabase/pgadmin-schema-diff:cli-0.0.5"
+	MigraImage      = "djrobstep/migra:3.0.1621480950"
+	PgmetaImage     = "supabase/postgres-meta:v0.53.1"
+	StudioImage     = "supabase/studio:0.22.08"
+	DenoRelayImage  = "supabase/deno-relay:v1.5.0"
+	ImageProxyImage = "supabase/imgproxy:v1.0.2"
```
Comment (sweatybridge) [inline internal/utils/misc.go:38]: Could you point me to the repo where this image is built from? Need to setup ecr and ghcr mirrors.

Comment (inian) [reply internal/utils/misc.go:38]: I already added ecr support Qiao - https://github.com/supabase/imgproxy/

Comment (inian) [reply internal/utils/misc.go]: mirrored to ghcr too and fixed tests

Comment (sweatybridge) [reply internal/utils/misc.go]: Nice! Thank you

**Review by sweatybridge — APPROVED** [review]

**Pull request merged**